notebooks/image_models/solutions/vertex_tuning_image_classification.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    %env TF_CPP_MIN_LOG_LEVEL=3
</pre><p>Level 3 doesn't print the error messages, I would like to see error messages though. So if you want to use this, maybe consider level=1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Changed to =1</p>